examples/leds_shell/Makefile

```diff
@@ -19,4 +19,9 @@ QUIET ?= 1
 USEMODULE += shell
 USEMODULE += periph_gpio
 
+# Prevents native program from segfault when gpio set/clear are used
+ifeq (native, $(BOARD))
```

Comment:
You also want to check for `native64`

Reply:
Till today, I never use `native64`- but it behave like `native` and crash without `gpio_periph_mock`.

Im not an exepert in `Makfile` but after reading few pages about or adding logical OR to `ifeq` [(for example)](https://stackoverflow.com/questions/7656425/makefile-ifeq-logical-or) i decided leave this "nasty" copy-paste code.